public/event.less

```diff
@@ -140,7 +140,7 @@
     padding: 0;
   }
 
-  .plugin-calendar-event-editor-dates {
+  .plugin-calendar-event-editor-dates .plugin-calendar-event-editor-date {

```

Comment:
This is my solution to #92. I am not 100% sure if this is the correct/best way to go, but it fixed it in my environment!